run_plotlc.py

```diff
@@ -31,7 +31,9 @@ def main():
 	parser.add_argument('--filters', '-f', type=str, nargs='*', default=None, choices=all_filters,
 		help='List of space delimited filters. If not provided will use all')
 	parser.add_argument('--offset', '-jd', type=float, default=2458800.0)
-	parser.add_argument('--subonly', help='Only show template subtracted data points.', action='store_true')
+	parser.add_argument('--subonly', type=bool, default=False, help='True or False')
```

Comment:
Do not revert this line, since the functionality has already been included and fixed